indico/types/utils.py

```diff
@@ -2,17 +2,18 @@
 import time
 from typing import TYPE_CHECKING
 
+from pydantic.alias_generators import to_snake
+
 from indico.errors import IndicoTimeoutError
 
 if TYPE_CHECKING:  # pragma: no cover
     from typing import Match, NoReturn, Optional, Union
 
-_cc_to_snake_re = re.compile(r"(?<!^)(?=[A-Z])")
 _snake_to_cc_re = re.compile(r"(.*?)_([a-zA-Z])")
```

Comment:
pydantic's `to_camel` behaves slightly differently that ours, in that it doesn't parse strings that start with `_`, hence why i did not replace it

Reply:
could you add a comment that says this, that way if we look at the code in the future we know why this choice was made?